backend/serializers/participant_serializer.py

```diff
@@ -18,8 +18,19 @@ class Meta:
 
     first_name = fields.Str(required=True, validate=[validate.Length(min=3, max=50)])
     last_name = fields.Str(required=True, validate=[validate.Length(min=3, max=50)])
-    email = fields.Email(required=True, validate=[validate.Length(min=3, max=200)])
-    github = fields.Str(required=True, validate=[validate.Length(min=1, max=39)])
+    email = fields.Email(required=True, validate=[validate.Email()])
+    github = fields.Str(
+        required=True, validate=[validate.Regexp("^[a-zA-Z0-9][a-zA-Z0-9._-]*$")]
+    )
+    # I am not sure why this one below doesn't work :(
+    # github = fields.Str(required=True, validate=[validate.Regexp("/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i")])
     hacknights = fields.Nested("HacknightSchema", exclude=("participants",), many=True)
-    phone = fields.Str(validate=[validate.Length(max=13)])
-    slack = fields.Str(validate=[validate.Length(max=21)])
+    phone = fields.Str(
+        validate=[
+            validate.Length(max=13),
+            validate.Regexp("^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$"),
```

Comment:
This is also highly complex/ Why the phone number is kept as a String? The representation (whether it starts with `+` or not, has some `-` inside) don't really matter so why store (or care about) them?
```suggestion
            validate.Regexp("^+?[0-9]{3}-[0-9]{3}-[0-9]{3}$"),
```

Reply:
It's kept as string as I thinkt that is the common approach. Int will remove preceding zeroes, we can't add `+` or `-` inside.